src/idl_gen_dart.cpp

```diff
@@ -339,6 +339,24 @@ class DartGenerator : public BaseGenerator {
     }
   }
 
+  static std::string EnumSize(const Type &type) {
+    switch (type.base_type) {
+      case BASE_TYPE_BOOL:
+      case BASE_TYPE_CHAR:
+      case BASE_TYPE_UTYPE:
+      case BASE_TYPE_UCHAR: return "1";
+      case BASE_TYPE_SHORT:
+      case BASE_TYPE_USHORT: return "2";
+      case BASE_TYPE_INT:
+      case BASE_TYPE_UINT:
+      case BASE_TYPE_FLOAT: return "4";
+      case BASE_TYPE_LONG:
+      case BASE_TYPE_ULONG:
+      case BASE_TYPE_DOUBLE: return "8";
+      default: return "1";
```

Comment:
This seems like it might not be following the formatting guidelines for this repo, but it's been a while so I'm not sure.

Reply:
I was trying to follow the formatting of the switch statement in the lines immediately preceding this one, and reading the style guide I _think_ I've got this right?

Reply:
Ah ok. In the codebases I usually work in this would not be allowed but it's better to keep it consistent here than to introduce differences. We can always change them all later if we really want.